selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/testrunner/JavaTestPluginJunit4Test.java

```diff
@@ -98,13 +93,13 @@ public void prepareTestProject() throws Exception {
     notifications.waitProgressPopupPanelClose();
     projectExplorer.quickExpandWithJavaScript();
 
-    runCompileCommandByPallete(compileCommand);
+    runCompileCommand();
```

Comment:
Could you, please, explain, why verification the command palette had been removed?

Reply:
Sometimes we have problems with command creation by ```TestCommandServiceClient``` and the created command is not visible in Command Pallete(see screenshot).
![org eclipse che selenium testrunner javatestplugintestngtest preparetestproject_time-1544721265452-millis](https://user-images.githubusercontent.com/7760565/50158662-68c0bb00-02dd-11e9-9e03-6a45766ea681.png)